src/main/assembly/jar-with-dependencies.xml

```diff
@@ -22,7 +25,7 @@
   <fileSets>
     <fileSet>
       <directory>${project.basedir}/src/main/config</directory>
-      <outputDirectory>/</outputDirectory>
+      <outputDirectory/>
```

Comment:
This doesn't have anything to do with my change but when I was running the "package" phase in Maven it complained that  "/" wasn't cross-platform. Specifying an empty string for "outputDirectory" (which this effectively does) seems to achieve the intended result of locating files at the root of the package, without the warning.

Reply:
Please remove.  Instead open separate pull requests for distinct changes.